Suggested edits related to Strings chapter (#1219)
hadley/r4ds: strings.qmd

### Letter variations

Working in languages with accents poses a significant challenge when determining the position of letters (e.g., with `str_length()` and `str_sub()`) as accented letters might be encoded as a single individual character (e.g., ü) or as two characters by combining an unaccented letter (e.g., u) with a diacritic mark (e.g., ¨).
For example, this code shows two ways of representing ü that look identical:

```{r}
u <- c("\u00fc", "u\u0308")
str_view(u)
```

> But both strings differ in length, and their first character is different:

Reviewer: ```suggestion
But both strings differ in length, and their first characters are different:
```

Base R string functions automatically use the locale set by your operating system.
This means that base R string functions do what you expect for your language, but your code might work differently if you share it with someone who lives in a different country.
To avoid this problem, stringr defaults to English rules by using the "en" locale and requires you to specify the `locale` argument to override it.
Fortunately, there are two sets of functions where the locale really matters: changing case and sorting.

The rules for changing cases differ among languages.

> For example, an i without a dot (i.e., ı) is capitalized differently in English or Turkish:

Reviewer: This is not quite correct since Turkish truly has two i's (two different letters in the alphabet).

> For example, an i without a dot (i.e., ı) is capitalized differently in English or Turkish:

Reviewer: ```suggestion
For example, Turkish has two i's: with and without a dot. Since they're two distinct letters, they're capitalized differently:
```
